src/main/java/ladder/controller/LadderGameController.java

```diff
@@ -0,0 +1,49 @@
+package ladder.controller;
+
+import ladder.domain.LadderGame;
+import ladder.domain.ResultCommand;
+
+public class LadderGameController {
+    private final InputController inputController;
+    private final ResultController resultController;
+
+    public LadderGameController(InputController inputController, ResultController resultController) {
+        this.inputController = inputController;
+        this.resultController = resultController;
+    }
+
+    public void run() {
+        try {
+            LadderGame ladderGame = inputController.initGame();
+            resultController.printLadder(ladderGame);
+            runGame(ladderGame);
```

Comment:
```suggestion
            ladderGame.run();
```
public 메소드라서 run()을 바로 호출할 수도 있는데, private 으로 한 번 더 랩핑하신 이유가 있을까요?